pkgs/development/python-modules/labgrid/default.nix

```diff
@@ -23,22 +24,22 @@
 
 buildPythonPackage rec {
   pname = "labgrid";
-  version = "0.4.1";
+  version = "23.0.1";
 
   src = fetchFromGitHub {
     owner = "labgrid-project";
     repo = "labgrid";
     rev = "v${version}";
-    sha256 = "0ih04lh1q3dysps4vfmk2rhqqrsimssadsxvbxdsnim2yihrrw47";
+    sha256 = "sha256-k/ReNr4GVrR/ghyzyQAo8NB6Eklno1/0fVMTyZk69SI=";
   };
 
+  nativeBuildInputs = [ setuptools-scm ];
+
   patches = [
-    # Pyserial within Nixpkgs already includes the necessary fix, remove the
-    # pyserial version check from labgrid.
-    ./0001-serialdriver-remove-pyserial-version-check.patch
+    ./0001-pyproject-unpin-and-use-upstream-pyserial.patch
```

Comment:
Please use sed/substituteInPlace to make updates easier. Reabsing this patch is useless manual labor.

Reply:
Done.